Add brand icons for global_guaguas

enriqueemarchena opened the pull request:
## Proposed change
Add brand icons for the custom integration `global_guaguas`. This integration provides bus data for the Canary Islands (Gran Canaria), specifically the Global company.

## Type of change

- [ ] Add a new logo or icon for a new core integration
- [ ] Add a missing icon or logo for an existing core integration
- [x] Add a new logo or icon for a custom integration (custom component)
  - [ ] I've added a link to my custom integration repository in the PR description
  - [ ] I've opened up a PR for my custom integration on the [Home Assistant Python wheels repository](https://github.com/home-assistant/wheels-custom-integrations)
- [ ] Replace an existing icon or logo with a higher quality version
- [ ] Replace an existing icon or logo after a branding change
- [ ] Removing an icon or logo

## Additional information

No external repository is linked publicly.

## Checklist

- [x] The added/replaced image(s) are **PNG**
- [x] Icon image size is 256x256px (`icon.png`)
- [x] hDPI icon image size is 512x512px for  (`icon@2x.png`)
- [x] Logo image size has min 128px, but max 256px, on the shortest side (`logo.png`)
- [x] hDPI logo image size has min 256px, but max 512px, on the shortest side (`logo@2x.png`)

frenck reviewed (changes requested):
Oh hi there @enriqueemarchena 👋 

Thanks for opening a pull request. However the CI is failing. Could you please take a look and make sure it passes? 

Thanks 👍 

../Frenck

enriqueemarchena commented:
@frenck Thank you very much for the feedback! I’ve fixed the issue and updated the branch. Please let me know if there’s anything else I should adjust.
